api/VideoPlaybackQuality.json

```diff
@@ -68,23 +55,10 @@
             "edge": {
               "version_added": "12"
             },
-            "firefox": [
-              {
-                "version_added": "42",
-                "version_removed": "73"
-              },
-              {
-                "version_added": "25",
-                "version_removed": "42",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "media.mediasource.enabled"
-                  }
-                ],
-                "notes": "Limited support to an allowed list of sites, for example YouTube, Netflix, and other popular streaming sites. The limitation was removed when Media Source Extensions was enabled by default in Firefox 42."
-              }
-            ],
+            "firefox": {
+              "version_added": "42",
+              "version_removed": "73"
```

Comment:
Confirmed using https://mdn-bcd-collector.appspot.com/tests/api/VideoPlaybackQuality/corruptedVideoFrames that it's not in the latest Firefox. I didn't look closer at what happened here.